translations/translation_nl.xml

```diff
@@ -33,24 +33,24 @@
 		<text name="COURSEPLAY_WAITING_TIME" 							 text="Wachttijd" />
 		<text name="COURSEPLAY_MINUTES" 								 text="%d min" />
 		<text name="COURSEPLAY_SECONDS" 								 text="%d sec" />
-		<text name="COURSEPLAY_UNIT_METER" 								 text="m" />            <!-- TODO: TRANSLATION! -->
+		<text name="COURSEPLAY_UNIT_METER" 								 text="m" />           
 		<text name="COURSEPLAY_COURSE_NAME" 							 text="Naam route:" />
 		<text name="COURSEPLAY_FOLDER_NAME" 							 text="Naam map:" />
 		<text name="COURSEPLAY_FILTER_COURSES" 							 text="Filter routes:" />
-		<text name="COURSEPLAY_BUTTON_RETURN" 							 text="Return" />					<!-- TODO: TRANSLATION! (This is the name of the Return Button) -->
-		<text name="COURSEPLAY_BUTTON_ESC" 								 text="ESC" />						<!-- TODO: TRANSLATION! (This is the short text of the Escape Button) -->
-		<text name="COURSEPLAY_CANCEL_WITH" 							 text="%s to Cancel" />				<!-- TODO: TRANSLATION! -->
-		<text name="COURSEPLAY_SAVE_COURSE_WITH" 						 text="%s to Save Course" />		<!-- TODO: TRANSLATION! -->
-		<text name="COURSEPLAY_SAVE_FOLDER_NAME_WITH" 					 text="%s to Save Folder" />		<!-- TODO: TRANSLATION! -->
-		<text name="COURSEPLAY_FILTER_COURSE_WITH" 						 text="%s to Filter Courses" />		<!-- TODO: TRANSLATION! -->
+		<text name="COURSEPLAY_BUTTON_RETURN" 							 text="Enter" />					
+		<text name="COURSEPLAY_BUTTON_ESC" 								 text="ESC" />						
+		<text name="COURSEPLAY_CANCEL_WITH" 							 text="%s om te annuleren" />				
+		<text name="COURSEPLAY_SAVE_COURSE_WITH" 						 text="%s om Route te bewaren" />		
```

Comment:
doesnt " COURSEPLAY_SAVE_COURSE_WITH"  mean " opslaan met"  ?
Its for the text in inputCoursename " save course with Return "